api/src/main/java/org/openmrs/ConceptMapType.java

```diff
@@ -11,18 +11,31 @@
 
 import org.codehaus.jackson.annotate.JsonIgnore;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
 /**
  * ConceptMapType are used to define relationships between concepts and concept reference terms e.g
  * IS_A or SAME_AS, BROADER_THAN
  *
  * @since 1.9
  */
+@Entity
+@Table(name = "concept_map_type")
 public class ConceptMapType extends BaseChangeableOpenmrsMetadata {
-	
+
 	private static final long serialVersionUID = 1L;
-	
+
```

Comment:
remove this space

Reply:
There is no extra space added @HerbertYiga on its some glitch with the version control which shows that line as deleted and then added again on its own. not sure how to remove that?